assets/src/edit-story/app/uploader/useUploader.js

```diff
@@ -23,8 +23,10 @@ import { useCallback } from '@wordpress/element';
  */
 import { useAPI } from '../../app/api';
 import { useConfig } from '../config';
+import { useMedia } from '../media';
 
-function useUploader() {
+function useUploader( { refreshLibrary = true } ) {
```

Comment:
You could default the argument to true while allowing no argument as:

```js
function useUploader( { refreshLibrary } = { refreshLibrary: true } ) {
```

But it's pretty ugly. It would mean [avoiding the empty object](https://github.com/google/web-stories-wp/pull/210/files#diff-c9756545ef6a994f3ba0091cf289a47fR46) though.

Reply:
Fixed in https://github.com/google/web-stories-wp/pull/210/commits/2db2350a61c2db751bea03c8ce8acb6384fae7ab